gradlew

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 
 #
-# Copyright � 2015-2021 the original authors.
+# Copyright © 2015-2021 the original authors.
```

Comment:
What's the actual change here? Is this good?

Reply:
changed the file encoding from cp1252 to utf-8